.circleci/config.yml

```diff
@@ -89,11 +92,13 @@ jobs:
               A up;
               A run monitoring up;
               A down;
-              cd "$PROTOCOL_DIR";
+              cd audius-protocol/;
```

Comment:
this env var is defined in the CI context, it's not passed through the ssh session which is why this was causing problems